interface/src/main/java/xsbti/api/Modifiers.java

```diff
@@ -10,7 +10,7 @@
 	private static final int LazyBit = 5;
 	private static final int MacroBit = 6;
 
-	private static final int flag(boolean set, int bit)
```

Comment:
In Java `final` method means that you can't [override or hide][1] it in the subclass. That concern is moot because `Modifier` is a final class. In addition, `flag` method being a private static class means it's used for its own implementation, but it's not externally observable. I'd say this is ok as is.

 [1]: http://www.coderanch.com/how-to/java/OverridingVsHiding